contracts/contracts.go

```diff
@@ -51,6 +51,27 @@ func ExampleToken(fungibleTokenAddr string) []byte {
 	return []byte(code)
 }
 
+// CustomizableExampleToken returns the ExampleToken contract with a custom Name.
+//
+// The returned contract will import the FungibleToken contract from the specified address.
```

Comment:
```suggestion
// The returned contract will import the FungibleToken interface from the specified address.
```

I could see this causing a little bit of confusion